src/models/Block.ts

```diff
@@ -0,0 +1,14 @@
+export interface IBlock {
+  _id: string;
+  height: number;
+  status: string;
+  anchor: number;
+  timestamp: Date;
+  root: string;
+  minter: string;
+  staked: number;
+  power: number;
+  voters: Array<string>;
+  votes: Map<string, number>;
```

Comment:
can we find a way to add names to structure? like:

```suggestion
  votes: Map<address: string, votes: string>; // I know this not work
```

maybe with `Record`? otherwise ` Map<string, string>` does not give us feedback what are the strings.

Reply:
Fixed